examples/with-aws-serverless/src/mock/users.ts

```diff
@@ -0,0 +1,28 @@
+const User1 = {
+  id: 'kimcoder',
+  name: 'kimcoder',
+  email: 'kimcoder@me.com',
+  photo:
+    'https://avatars2.githubusercontent.com/u/2926726?s=460&u=a20c1aa215d7e59f9d61cff54eb3c158db922a57&v=4',
+}
+
+const User2 = {
+  id: 'henrycavill',
+  name: 'henrycavill',
+  email: '',
+  photo:
+    'https://scontent-ssn1-1.cdninstagram.com/v/t51.2885-19/s320x320/10299788_1632819743635060_1564818683_a.jpg?_nc_ht=scontent-ssn1-1.cdninstagram.com&_nc_ohc=pF7qy8YcW2MAX8g28YN&oh=05338017fe0bd73aad99591a43e3312b&oe=5EE71688',
```

Comment:
These should not reference instagram urls as they could break and or infringe on copyright. You can potentially replace them with unsplash images: http://unsplash.com/